azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/Frontend.java

```diff
@@ -13,17 +13,17 @@
 import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
 
 /**
- * An immutable client-side representation of an HTTP load balancing rule.
+ * An immutable client-side representation of a load balancer frontend.
  */
 public interface Frontend extends
     Wrapper<FrontendIPConfigurationInner>,
     ChildResource,
     HasLoadBalancingRules {
 
     /**
-     * @return true if the front end is Internet-facing, i.e. it has a public IP address associated with it
+     * @return true if the frontend is public, i.e. it has a public IP address associated with it
      */
-    boolean isInternetFacing();
+    boolean isPublic();
 
```

Comment:
Some errors returned by network RP uses `internal and public` texts to distinguish between two different type of load balancers - so it's should be ok to have that separation in fluent layer though in swagger there is no different types for different LB types.